packages/mobile/src/helpers/date/format-date/format-date.helper.ts

```diff
@@ -0,0 +1,12 @@
+import dayjs from 'dayjs';
+import { DateFormat } from '~/common/enums/enums';
+
+const formaTotDateTime = (date: Date) => {
+  return dayjs(date).format(DateFormat.dateTime);
+};
+
+const getSellerTimeZone = (date: Date) => {
+  return dayjs(date).format(DateFormat.timeZone);
```

Comment:
This will always return your local time zone. There is no sense in this

Reply:
date would come from server, with time zone that recorded there

Reply:
removed